Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -886,7 +890,7 @@ sendSolutionMessageNextTask=Meldung \u00FCber Probleml\u00F6sung an nachfolgende
 sendSolutionMessage=Meldung \u00FCber Probleml\u00F6sung senden
 sendSolutionMessageForAll=Meldung \u00FCber Probleml\u00F6sung f\u00FCr alle Schritte senden
 sendSolutionMessageForSingle=Meldung \u00FCber Probleml\u00F6sung f\u00FCr den aktuellen Schritt senden
-separateStructure=Erweiterter Stukturbaum
+separateStructure=Erweiterter Stukturbaum (experimentell)
```

Comment:
Multiple errors where reported about the metadata editor that only occur when the `separateStructure` flag is set during workflow configuration (#4700 , #4395 etc.). Since the "Extended View"/"Separate structure" is not widely used, yet, I wanted to mark it accordingly (like beta state or somethink like that). 

Granted, it does not have anything to do with fixing the correction workflow, I but I thought sneaking in one little unrelated comment wouldn't hurt. Would you propose to create a separate pull request for it?

Reply:
I wouldn't make that change _at all._ The physical structure tree is sometimes in use, or we tell customers that they have this option to use it if they want to realize very specific links between the physical and logical levels. If there are bugs, they should be fixed. But “experimental” now sounds _very immature_, and that's not true. Otherwise, we should approach it differently and only make it available _at all_ via a switch in the `config.properties` file, in the “dangerous and broken things” section. And yes, I would suggest making _an issue_ for discussing this separately.